transcom/mymove
# MB-15588 Add eslint jsx-a11y to the src linter

## [Jira ticket](https://dp3.atlassian.net/browse/MB-15588) for this change

## Summary

Add all of the recommended `jsx-a11y` lint rules to the `eslintrc`.
CRA comes with a set of `jsx-a11y` rules configured, but to enable all of them, the `jsx-a11y` plugin needs to be extended, which I have done here. 

https://www.freecodecamp.org/news/react-accessibility-tools-build-accessible-react-apps/
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y

## How to Test

1. Open up a jsx file in VSCode or whichever text editor you have configured for linting. 
2. Add `<iframe />` to the return of the component. 
3. Your editor should highlight the error in red with reference to a jsx-a11y rule being broken. If not, you can also run `yarn run lint:production` in your terminal of choice, and that should flag errors as well. 

## Verification Steps for Author

These are to be checked by the author.
